src/main/java/io/seqera/tower/cli/commands/labels/AddLabelsCmd.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2021, Seqera Labs.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * This Source Code Form is "Incompatible With Secondary Licenses", as
+ * defined by the Mozilla Public License, v. 2.0.
+ */
+
+package io.seqera.tower.cli.commands.labels;
+
+import io.seqera.tower.ApiException;
+import io.seqera.tower.cli.commands.computeenvs.AbstractComputeEnvCmd;
+import io.seqera.tower.cli.commands.global.WorkspaceOptionalOptions;
+import io.seqera.tower.cli.exceptions.ShowUsageException;
+import io.seqera.tower.cli.exceptions.TowerException;
+import io.seqera.tower.cli.responses.Response;
+import io.seqera.tower.model.CreateLabelRequest;
+import io.seqera.tower.model.CreateLabelResponse;
+import picocli.CommandLine;
+
+import javax.annotation.Nullable;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+@CommandLine.Command(
+        name = "add",
+        description = "Add new label"
+)
+public class AddLabelsCmd extends AbstractLabelsCmd {
+
+    @CommandLine.Option(names = {"-n", "--name"}, description = "Label name.", required = true)
+    public String labelName;
+
+    @CommandLine.Option(names = {"-w", "--workspace"}, description = "Workspace numeric identifier", required = false)
+    @Nullable
+    public Long workspaceId;
```

Comment:
Use `WorkspaceOptionalOptions` mixin and then the method `workspaceId` to resolve the workspace, this allows to use same parameter to pass the workspace using the name or the id.

See a usage example here:
http://github.com/seqeralabs/tower-cli/blob/70c30c586a05cfc44819aa0cfd2204291f95e7b4/src/main/java/io/seqera/tower/cli/commands/secrets/AddCmd.java#L43-L43